src/components/CollapsibleSection/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {View, TouchableOpacity} from 'react-native';
 import Collapsible from './Collapsible';
 import Text from '../Text';
-import {propTypes, defaultProps} from './CollapsibleSectionPropTypes';
+import propTypes from './CollapsibleSectionPropTypes';
```

Comment:
I can't find where these are used in any other file. Can you please just move them into this component to clean it up a little further?